views/membership/index.hbs

```diff
@@ -5,14 +5,13 @@
 		<p class="mt-3 mb-3">Displaying all <b>{{membership.length}}</b> matching Memberships on record</p>
 		<a class="btn btn-light me-3" href="new/">+ Add Membership</a>
 		<a class="btn btn-success" href="import">Import Membership</a>
-
-		<div class="row mb-3 row-cols-1 row-cols-md-2 d-none">
-			<div class="col mb-3">
+		<div class="row row-cols-1 row-cols-md-3">
+			<div class="col my-3 offset-4">
```

Comment:
Not sure why you'd do this 🧐

This may be nicer as a centered block element, with 50% width on desktop and full width on mobile